src/hooks/transfer/useTransfer.ts

```diff
@@ -116,12 +116,9 @@ export function useTransfer(selectUnit: Ref<string>, decimal: Ref<number>, fn?:
       }
 
       const receivingAddress = isValidEvmAddress(toAddress) ? toSS58Address(toAddress) : toAddress;
-
-      watchEffect(async () => {
```

Comment:
@sirius651 
I've removed `watchEffect` here. Please advise if it causes any unexpected behavior.